package.json

```diff
@@ -27,6 +27,8 @@
     "@babel/eslint-plugin": "^7.14.5",
     "@sheerun/mutationobserver-shim": "^0.3.3",
     "@testing-library/jest-dom": "^5.3.0",
+    "expect": "^28.1.3",
+    "pretty-format": "^28.1.3",
```

Comment:
This shouldn't be needed.

Reply:
These packages are causing the `npm ci` part of the pipeline to fail since they don't support node 12.  If that won't cause issues with the actual release, we could merge with the broken pipeline. I just don't know enough to make that call

Reply:
Looks like even pinning them here isn't sufficient since remote execution has them too as dependencies

Reply:
(I see now that I'm pinning them wrong anyway)